quodlibet/po/he.po

```diff
@@ -5296,7 +5296,7 @@ msgstr "שמירת שינויים בתגים ללא בקשת אישור בעת 
 #: quodlibet/qltk/prefs.py:608
 msgid ""
 "A set of separators to use when splitting tag values in the tag editor. The list "
-"is space-separated"
+"is space-separated."
 msgstr ""
 "סדרת תווי הפרדה לשימוש בעת פיצול ערכים בעורך התגים. ערכי הרשימה מופרדים בפסיקים"
```

Comment:
Looks good, though I think this (translated) line should also end in a period.

Reply:
Probably yes, but as Hebrew is right-to-left, I didn't want to mess up the translation. I think it is safer to leave as-is.

Reply:
Oh.. good point. Totally didn't think of that.

Reply:
Best to just leave the original string as is in the future, so it will be marked as "fuzzy" by gettext and the translator sees that the string needs an update